package com.revature.exceptions;

public class NotFoundException extends RuntimeException {
    public NotFoundException() { super(); }
    public NotFoundException(Throwable cause) { super(cause); }
}
